src/hooks/useCreateDonation.tsx

```diff
@@ -153,6 +154,18 @@ export const useCreateDonation = () => {
 				setDonationSaved(true);
 				return id;
 			} catch (e) {
+				postRequest('/api/donation-backup', true, {
+					chainId: transaction?.chainId!,
+					txHash: transaction?.hash,
+					amount: amount,
+					token,
+					projectId,
+					anonymous,
+					nonce: transaction?.nonce,
+					chainvineReferred,
+					walletAddress: transaction?.from,
+					symbol: token.symbol,
```

Comment:
Is it possible to include some information of the authentication token? e.g. expiration data, nonce, jwt, ...

Reply:
@aminlatifi I can add the user's token.